tests/mock_jsons/3.9.41/build_test-build-123.json

```diff
@@ -3,168 +3,103 @@
         "strategy": {
-            "type": "Custom",
             "customStrategy": {
                 "env": [
                     {
                         "name": "REACTOR_CONFIG",
                         "valueFrom": {
                             "configMapKeyRef": {
-                                "name": "reactor-config-map",
-                                "key": "config.yaml"
+                                "key": "config.yaml",
+                                "name": "reactor-config-map"
                             }
                         }
                     },
                     {
                         "name": "USER_PARAMS",
-                        "value": "{\"arrangement_version\": 6, \"base_image\": \"osbs-test/base:1.0.public.distscope\", \"build_imagestream\": \"osbs-rcm-buildroot:staged\", \"build_json_dir\": \"/usr/share/osbs/\", \"build_type\": \"orchestrator\", \"component\": \"osbs-test-sandwich-docker\", \"customize_conf\": \"orchestrator_customize.json\", \"git_branch\": \"add-repo-url\", \"git_ref\": \"e411351\", \"git_uri\": \"git://git.example.com/rpms/osbs-test-sandwich-docker\", \"image_tag\": \"autobot-brew-stage-osbs/osbs-test-sandwich-docker:osbs-test-1.0-rhel-7-docker-candidate-14109-20181029083106\", \"imagestream_name\": \"osbs-test-add-repo-url\", \"koji_target\": \"osbs-test-1.0-rhel-7-docker-candidate\", \"name\": \"osbs-test-sandwich-docker-add-repo-url-46fc0\", \"platforms\": [\"x86_64\"], \"reactor_config_map\": \"reactor-config-map\", \"trigger_imagestreamtag\": \"osbs-test-base:1.0.public.distscope\", \"user\": \"autobot-brew-stage-osbs\", \"yum_repourls\": [\"files.example.com/osbs/repos/osbs-rhel7-latest.repo\"]}"
-                    },
-                    {
-                        "name": "OPENSHIFT_CUSTOM_BUILD_BASE_IMAGE",
-                        "value": "buildroot:latest"
+                        "value": "{\"arrangement_version\": 6, \"base_image\": \"fedora:latest\", \"build_imagestream\": \"osbs-rcm-buildroot:staged\", \"build_json_dir\": \"/usr/share/osbs/\", \"build_type\": \"worker\", \"component\": \"chromium\", \"customize_conf\": \"worker_customize.json\", \"git_branch\": \"master\", \"git_commit_depth\": 1, \"git_ref\": \"HEAD\", \"git_uri\": \"https://github.com/TomasTomecek/docker-hello-world.git\", \"image_tag\": \"mlangsdo/chromium:none-16225-20191009203233-x86_64\", \"imagestream_name\": \"myproject-hello-world\", \"name\": \"docker-hello-world-master-bd212\", \"orchestrator_deadline\": 4, \"platform\": \"x86_64\", \"reactor_config_map\": \"reactor-config-map\", \"release\": \"test-build-123\", \"scratch\": true, \"trigger_imagestreamtag\": \"fedora:latest\", \"user\": \"mlangsdo\", \"worker_deadline\": 3}"
                     }
                 ],
                 "exposeDockerSocket": true,
                 "from": {
                     "kind": "DockerImage",
-                    "name": "buildroot:latest"
+                    "name": "example.com"
```

Comment:
This looks suspicious

Reply:
It happens because of the sanitization we perform: any string containing "redhat.com" gets replaced with "example.com". Think that's good enough?

Reply:
it was changed from `buildroot:latest ` so it looks suspicious, but if this is expected I'm fine with that

Reply:
Yes, original string from the server here contained "redhat.com". "buildroot:latest" was most likely a replaceement string itself, for the same reason.

Reply:
I went ahead and replaced this with some regex to do the right thing.